metaflow/plugins/airflow/airflow_utils.py

```diff
@@ -401,9 +419,27 @@ class MetaflowKubernetesOperator(KubernetesPodOperator):
         This is later useful in the `Workflow.compile` where the operator's `expand` method is called and we are able to retrieve the xcom value.
         """
 
-        def __init__(self, *args, mapper_arr=None, **kwargs) -> None:
+        template_fields = KubernetesPodOperator.template_fields + (
+            "metaflow_pathspec",
+            "metaflow_run_id",
```

Comment:
can we also set - `metaflow_task_id`, `metaflow_attempt_id`, `metaflow_step_name`, `metaflow_flow_name`?

Reply:
Done